deploy/README.md

```diff
@@ -44,10 +44,10 @@ The following table displays the tested Kubernetes and Helm versions.
 
 Name | Version
 -------- | -----
-K8s with EKS | 1.14<br/>1.15<br/>1.16<br/>1.17
+K8s with EKS | 1.15<br/>1.16<br/>1.17<br/>1.18
 K8s with Kops | 1.16<br/>1.17<br/>1.18
 K8s with GKE | 1.15<br/>1.16
```

Comment:
1.15 version is only supported for release 1.3. So, will it be okay to clearly mention that or will this work?

Reply:
This is a `v1.3` documentation update (I'm merging into the `release-v1.3` branch).